src/Engine/Party.cpp

```diff
@@ -1199,7 +1199,7 @@ bool Party::AddItemToParty(ItemGen *pItem) {
     if (v5) {
         auto texture = assets->GetImage_ColorKey(v5, render->teal_mask_16);
         v8 = 0;
-        uint current_player = uActiveCharacter;
+        uint current_player = std::min(1u, uActiveCharacter);
```

Comment:
std::min? Why? As I understand logic there was cycling characters from active one and if there is no space then try next one and rollback to first chars if first one wasn't first... And now you are placing items starting from first character which is inconvinient...

If you wanted to overcome our uActiveCharacter = 0 issue, then you can set uActiveCharacter = 1 if it was 0. Also I'll deny any billboard/buildings interactions if it is uActiveCharacter = 0 in pr351 as was proposed in discord to workaround inability of many functions which couldn't handle uActiveCharacter = 0...

Reply:
Good spot. I intended std::max